manifests/0000_21_openshift-controller-manager-operator_07_deployment.yaml

```diff
@@ -36,6 +36,8 @@ spec:
         volumeMounts:
         - mountPath: /var/run/configmaps/config
           name: config
+        - mountPath: /var/run/secrets/serving-cert
+          name: serving-cert
```

Comment:
@deads2k fyi from what i saw at least the kube-controller operator is not mounting this properly.

Reply:
> @deads2k fyi from what i saw at least the kube-controller operator is not mounting this properly.

ooops